src/index.ts

```diff
@@ -3,6 +3,6 @@ export * from "./interfaces";
 
 export { runTaskList, runTaskListOnce } from "./main";
 export { run, runOnce, runMigrations } from "./runner";
-export { Logger, LogFunctionFactory, consoleLogFactory } from "./logger";
+export { Logger, LogFunctionFactory, consoleLogFactory, defaultLogger } from "./logger";
```

Comment:
Prettier want to expand this line now...don't know if that's in your CI.  I will leave this unless you tell me otherwise or CI fails.

Reply:
Should have waited.